amoro-ams/amoro-ams-dashboard/src/language/en.ts

```diff
@@ -198,8 +198,9 @@ export default {
   totalDeleteFiles: 'delete-files',
   branches: 'branches',
   tags: 'tags',
+  consumer: 'consumer',
   nothingToShow: 'Nothing to show',
-  filterBranchesOrTags: 'Filter branches/tags',
+  filterBranchesOrTags: 'Filter branches/tags/consumer',
```

Comment:
filterBranchesOrTagsOrConsumers: 'Filter branches/tags/consumers',

Related `filterBranchesOrTags` in the code should be replaced by `filterBranchesOrTagsOrConsumers`